generators/entity-server/templates/src/main/resources/config/liquibase/fake-data/table.csv.ejs

```diff
@@ -138,8 +138,8 @@ for (lineNb = 1; lineNb <= numberOfRows; lineNb++) {
         }
         if (fields[idx].fieldIsEnum) {
             if (fields[idx].fieldValues.length !== 0) {
-                const enums = fields[idx].fieldValues.split(',');
-                data = enums[this.faker.random.number(enums.length-1)];
+                const enumValues = getEnumValuesWithCustomValues(fields[idx].fieldValues);
```

Comment:
Not 100% sure but these 2 lines causes some troubles with fake data.

This CSV can't be imported into the application.

```
id;my_field_a;my_field_b;my_field_c
1;AAA;BBB;aaa_aaa
2;BBB;aaa_aaa;aaa_aaa
3;BBB;aaa_aaa;aaa_aaa
4;BBB;BBB;b and b
5;AAA;BBB;b and b
6;AAA;aaa_aaa;aaa_aaa
7;BBB;BBB;b and b
8;BBB;aaa_aaa;aaa_aaa
9;BBB;aaa_aaa;aaa_aaa
10;BBB;BBB;b and b
```

This one can:

```
id;my_field_a;my_field_b;my_field_c
1;AAA;BBB;AAA
2;BBB;AAA;AAA
3;BBB;AAA;AAA
4;BBB;BBB;BBB
5;AAA;BBB;BBB
6;AAA;AAA;AAA
7;BBB;BBB;BBB
8;BBB;AAA;AAA
9;BBB;AAA;AAA
10;BBB;BBB;BBB
```

So maybe try to use the "key" instead of the "value" here?

Reply:
You can play with:
- KO -> https://github.com/pascalgrimaud/jh-issue-11588
- OK -> https://github.com/pascalgrimaud/jh-issue-11588/tree/fix-enums

Just launch ./mvnw and go to http://localhost:8080/field-test-enum-with-value

Reply:
I love you man

Reply:
Should be fixed in next line.